app/src/main/java/io/homeassistant/companion/android/database/sensor/SensorWithAttributes.kt

```diff
@@ -14,7 +14,17 @@ data class SensorWithAttributes(
     val attributes: List<Attribute>
 ) {
     fun toSensorRegistration(): SensorRegistration<Any> {
-        val attributes = attributes.map { it.name to it.value }.toMap()
+        val attributes = attributes.map {
+            val attributeValue = when (it.valueType) {
+                "" -> ""
```

Comment:
If we don't default we should never get this, we have this for state because there are times that there is no valid state.